torch/ao/quantization/backend_config/backend_config.py

```diff
@@ -69,14 +66,17 @@ class ObservationType(Enum):
 @dataclass
 class DTypeWithConstraints:
     """
-    Config for specifying additional constraints for a given dtype, such as quantization value
-    ranges and scale value ranges, to be used in :class:`~torch.ao.quantization.backend_config.DTypeConfig`.
+    Config for specifying additional constraints for a given dtype, such as quantization
+    value ranges, scale value ranges, and fixed quantization params, to be used in
+    :class:`~torch.ao.quantization.backend_config.DTypeConfig`.
     """
     dtype: Optional[torch.dtype] = None
     quant_min_lower_bound: Union[int, float, None] = None
     quant_max_upper_bound: Union[int, float, None] = None
     scale_min_lower_bound: Union[int, float, None] = None
     scale_max_upper_bound: Union[int, float, None] = None
+    fixed_scale: Optional[float] = None
```

Comment:
how about something like `scale_exact_match` and `zero_point_exact_match`, to be consistent with the naming of the other attributes?